include/boost/numeric/odeint/stepper/bulirsch_stoer_dense_out.hpp

```diff
@@ -129,6 +130,7 @@ class bulirsch_stoer_dense_out {
             else
                 m_cost[i] = m_cost[i-1] + m_interval_sequence[i];
             m_coeff[i].resize(i);
+            m_facmin_table[i] = pow BOOST_PREVENT_MACRO_SUBSTITUTION( STEPFAC3 , 1.0 / static_cast< value_type > ( 2*i+1 ) );
```

Comment:
This fix might have been to rushed, now the computation of ```facmin_table[0]``` accesses ```interval_sequence[-1]```. I suggest to move this computation into the if clause above and set ```facmin_table[0] = 1```. Note that this value will (should...) never be used and one could therefore use one less entry in ```facmin_table```, but the other tables like ```cost``` also have this additional entry for ```k=0``` which is not used so this would be consistent.